src/main.rs

```diff
@@ -68,7 +68,6 @@ fn run() -> Result<bool> {
             .set_location_level(LevelFilter::Debug)
             .set_target_level(LevelFilter::Off)
             .set_thread_level(LevelFilter::Off)
-            .set_level_padding(simplelog::LevelPadding::Left)
```

Comment:
i like the way it is right now and this isn't a clear improvement, so let's drop this change

the uid one looks good!

Reply:
The main reason I removed this is because the padding isn't applied to both sides, so it looks a bit weird to me. The other is because running with -vv, debug logs are printed as [DEBUG], not [ DEBUG], so the two types of logs don't match. If you want it removed no worries. :)

Reply:
I'm kinda mimicking how cargo does it because I like it🙃 
![cargo doc output sample](https://owo.whats-th.is/6uFXjYU.png)

I don't *really* mind either way

Reply:
No worries about it, I'll revert it.